spec/delta_changes_spec.rb

```diff
@@ -103,5 +103,24 @@
       user.score = '5'
       expect(user.delta_changes).to eq({})
     end
+
+    context 'when a tracked column is changed multiple times in a single db transaction' do
+      context 'and the final value is the same as the original' do
+        context 'and delta_changes is reset between each change' do
+          it 'should be filled after each change is applied' do
```

Comment:
can leave all the context in a comment if it is important
```suggestion
    it 'does not track extra changes when changing back and forth from original value'
```